### DISCUSSION TITLE
zepeto-multiplay-example에서 모바일 접속 시 오류

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#77] [5. Q&A]
zepeto-multiplay-example에서 QR 코드 생성하여 모바일로 접속하면, 아래와 같이 첫 화면만 뜨고 캐릭터나 사용자 입력 버튼이 보이질 않습니다. 화면 변화가 전혀 없는 상태로 접속되는데, 수정 방법 좀 알려주세요. PC 버젼에서는 잘 됩니다.
![image](https://user-images.githubusercontent.com/95363110/145815399-1f70496c-2c59-441b-9e7e-5a965610f922.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 위 현상이 최신 월드에서도 계속 나타나는지 확인 부탁드립니다.
감사합니다.
